### PR TITLE
662 convert to date

### PR DESCRIPTION
**Pull request by jonathonmellor**

### CODE REVIEW

**Comment by jonathonmellor** [comment]
Derive 13

**Comment by foster999** [comment]
Looks like tests are failing on the schema type for columns - expected is showing StringType for both I think

**Comment by jonathonmellor** [comment]
@foster999 I incorrectly resolved a merge locally (had split and date functions with the same line of code, so test failed). Should be fixed now and ready for review.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (foster999) [inline .vscode/settings.json:1]: Please could we drop this file (will be handled by .gitignore in future) :)

File: cishouseholds/derive.py [hunk 2]
```diff
@@ -144,6 +144,31 @@ def assign_column_uniform_value(df: DataFrame, column_name_to_assign: str, unifo
...
+    """
+    Assign a column with a TimeStamp to a DateType
+    From households_aggregate_processes.xlsx, derivation number 13.
+    Parameters
+    ----------
+    df
+    column_name_to_assign
+        Name of column to be assigned
+    reference_column
+        Column of TimeStamp type to be converted
```
Comment (foster999) [inline cishouseholds/derive.py:157]: Minor suggestion - could indicate that this is the "Name of column..." for clarity

**Review by foster999 — APPROVED** [review]
Other than the two comments, happy for this to be merged 👍🏻

File: cishouseholds/derive.py (hunk 2 above)
Comment (jonathonmellor) [reply cishouseholds/derive.py]: Should be done.

**Comment by jonathonmellor** [comment]
Removed `/.vscode/` from tracking. May be still in history, but non-sensitive.

**Pull request merged**